src/app/components/homepage/chambres/bedroom/bedroom.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="room-widget">
+  <!--TODO : carrousel -->
+  <div class="room-carrousel">
+    <img src="assets/room_place_holder.jpg" class="name-img" />
+  </div>
+  <div class="room-details">
+    <div class="room-header">
+      <div class="room-display-row">
+        <p class="title-widget-format room-title">{{ bedroomName }}</p>
+        <p class="title-widget-format price">{{ price }}</p>
+      </div>
+      <p class="subtitle-widget-format person-max">
+        {{ numberPersonMax }} {{ "CHAMBRES.COMPONENT.PEOPLE" | translate }}
+      </p>
+      <p class="subtitle-widget-format surface">{{ numberSurface }} m²</p>
+    </div>
+    <div class="room-more-details">
+      <div class="room-details-left">
+        <p class="text-widget-format room-bed-details">{{ bedType }}</p>
+        <div class="room-description-container">
+          <p class="text-widget-format room-description">
+            {{ shortDescription }}
+          </p>
+        </div>
+      </div>
+      <div class="room-details-right">
+        <p class="text-widget-format room-assets-title">
+          {{ "CHAMBRES.COMPONENT.CARACTERISTICS" | translate }} :
+        </p>
+        <div class="text-widget-format room-assets-list">
+          @for(asset of assets; track asset){
+          <p class="text-widget-format room-asset">{{ asset }}</p>
+          }
+        </div>
+      </div>
```

Comment:
Add a check on `assets.length > 0` to avoid displaying unused text:
![image](https://github.com/tjouliab/le-grenier-front/assets/108274561/b37c81ec-0ab3-4fef-ad8f-f9ca1a695374)